docs/examples.html

```diff
@@ -76,6 +76,35 @@
     </div>
   </section>
 
+  <section id="multiple_max" class="row">
+    <div class="col-md-4">
+      <h1>Limited Multiple select boxes</h1>
+      <p>Select2 multi-value select boxes can set restrictions regarding the maximum number of options selected.
+        The select below is declared with the <code>multiple</code> attribute with <code>maxSelectionLength</code> in the select2 options</p>
+
+      <p>
+        <select class="js-example-basic-multiple-limit js-states form-control" multiple="multiple"></select>
+      </p>
+    </div>
+    <div class="col-md-8">
+      <h2>Example code</h2>
+
+      <pre data-fill-from=".js-code-multiple-limit"></pre>
+
+<script type="text/x-example-code" class="js-code-multiple-limit">
+$(".js-example-basic-multiple-limit").select2({
+  maximumSelectionLength: 2
+  });
+
+<select class="js-example-basic-multiple" multiple="multiple">
```

Comment:
This can probably be dropped, there isn't anything special about the HTML for this, so I don't think it needs to be reinforced.